src/sections/Community/Handbook/connect.js

```diff
@@ -16,42 +23,96 @@ const Connect= () => {
-            <p><span className="highlight">#wasm</span> - For discussions on web assembly, rust, data plane filters</p>
-            <p><span className="highlight">#websites</span> - For discussions related to layer5.io site.</p>
-            <p><span className="highlight">#workshops</span> - Q-A for workshops given at KubeCon, DockerCon, Velocity, O’Reilly Live Training</p>
+            <p>Join our <a href="http://slack.layer5.io/">Slack Workspace</a> “introduce yourself” and interact with the community. There are different channels in the workspace, to engage in, just search for:</p>
+            <div style={{paddingLeft: "40px"}}>
+              <p><a href="https://layer5io.slack.com/archives/CDM0ACDM5">#general</a> - For workspace-wide communication and announcements.</p>
+              <p><a href="https://layer5io.slack.com/archives/CFGG6U10E">#meshery</a> - For discussion on topics in and around Meshery.</p>
+              <p><a href="https://layer5io.slack.com/archives/C017NA80S66">#meshery-adapters</a> - Discuss topics related to meshery-adapters.</p>
+              <p><a href="https://layer5io.slack.com/archives/C011VPREG2Z">#meshery-ci</a> - For discussion on topics related to meshery-ci.</p>
+              <p><a href="https://layer5io.slack.com/archives/C010LFFGFFA">#meshery-meshsync</a> - For discussions on issues related to mesh sync.</p>
+              <p><a href="https://layer5io.slack.com/archives/C0130BQ4L3T">#mesheryctl</a> - For discussions on Meshery’s CLI</p>
+              <p><a href="https://layer5io.slack.com/archives/C019426UBNY">#newcomers</a> - For welcoming first-time contributors and community newcomers.</p>
+              <p><a href="https://layer5io.slack.com/archives/CVBGPTFJN">#performance</a> - For discussions related to measuring and evaluating Service Mesh Performance.</p>
+              <p><a href="https://layer5io.slack.com/archives/C010H0HE2E6">#smi</a> - For discussions related to Meshery and Service Mesh Interface (SMI).</p>
+              <p><a href="https://layer5io.slack.com/archives/C012UMS2MCM">#wasm</a> - For discussions on web assembly, rust, data plane filters.</p>
+              <p><a href="https://layer5io.slack.com/archives/C015QJKUMPU">#websites</a> - For discussions related to the UI/UX of <a href="http://layer5.io/">Layer 5</a> project sites.</p>
+            </div>
+
+            <h2>Discussion Forum</h2>
+            <p>Join us on our service mesh community's <a href="https://discuss.layer5.io">Discussion Forum</a></p>
+
+            <h2>Social Media</h2>
+            <p>Layer5 is dedicated to providing the most efficient Service mesh management tool while growing a healthy open-source community. To know more about layer5 and connect with the community, follow and engage with us on our social media accounts.</p>
+            <div style={{paddingLeft: "40px"}}>
+              <p style={{display: "flex", alignItems: "flex-end"}}>
```

Comment:
Can you avoid using inline styles here? Since all the following icons have the same style, you can give them a className and move the styles to the Handbook stylesheet, that way we can avoid repetition here.

Reply:
Done @debo19!! Thank you so much for your patience.